core/storage/suggestion/gae_models.py

```diff
@@ -313,6 +313,45 @@ def get_translation_suggestions_in_review_with_exp_id(
             cls.target_id == exp_id
         )).fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_multiple_translation_suggestions_in_review(
+            cls, suggestion_ids: List[str]
+    ) -> List[Optional['GeneralSuggestionModel']]:
+        """Returns translation suggestions which are in review from multiple
+        thread ids.
+
+        Args:
+            suggestion_ids: list(str). Suggestion ids of translations that are
+                in review.
+
+        Returns:
+            list(SuggestionModel|None). A list of translation suggestions in
+            review with given suggestion ids.
+        """
+        return GeneralSuggestionModel.get_multi(suggestion_ids)
```

Comment:
Shouldn't we filter by in review status here?

Reply:
The function name and the description was misleading—it actuallay returns multiple suggestions for a given list of suggestion ids. Updated the functiona name and the description.

Reply:
Ah, gotcha.